Steps4Impact/Challenge/CreateTeam.swift

```diff
@@ -75,9 +92,41 @@ class CreateTeamViewController: ViewController {
       $0.height.equalTo(1)
     }
 
+    view.addSubview(teamPhotoTextField) {
+      $0.leading.trailing.equalToSuperview().inset(Style.Padding.p32)
+      $0.top.equalTo(seperatorView.snp.bottom).offset(Style.Padding.p16)
+    }
+
+    imgButton.contentMode = .scaleAspectFit
+    imgButton.layer.cornerRadius = 0.5 * Style.Size.s128
+    view.addSubview(imgButton) { (make) in
+      make.centerX.equalToSuperview()
+      make.top.equalTo(teamPhotoTextField.snp.bottom).offset(Style.Padding.p32)
+      make.height.width.equalTo(Style.Size.s128)
+    }
+
+    stackView.axis  = NSLayoutConstraint.Axis.horizontal
```

Comment:
you can just use the last part of the enum here since the type is inferred so `stackView.axis = ..horizontal ` - same for other enums below